src/co_pdo.c

```diff
@@ -575,6 +578,37 @@ int co_pdo_timer (co_net_t * net, os_tick_t now)
       }
    }
 
+   /* Check for RPDOs with event timer (deadline monitoring) */
```

Comment:
Should this loop over pdo_rx instead of pdo_tx?

Reply:
Almost certainly... Let me just see how on earth I got this to work.